<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="gameIconsScrollArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>469</width>
             <height>492</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addQuestsPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Add a quest list</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListAdd"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeQuestsPushButton">
            <property name="toolTip">
             <string>Delete selected quest list</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListRemove"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="questsMetadata">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border-style: none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="actionExport_all"/>
     <addaction name="actionExport_selected"/>
    </widget>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_new_quest_list_2"/>
    <addaction name="actionDelete_selected_quest_list_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_new_quest_list">
   <property name="text">
    <string>Add new quest list</string>
   </property>
  </action>
  <action name="actionDelete_selected_quest_list">
   <property name="text">
    <string>Delete selected quest list</string>
   </property>
  </action>
  <action name="actionAdd_new_quest_list_2">
   <property name="text">
    <string>Add new quest list</string>
   </property>
  </action>
  <action name="actionDelete_selected_quest_list_2">
   <property name="text">
    <string>Delete selected quest list</string>
   </property>
  </action>
  <action name="actionExport_all">
   <property name="text">
    <string>Export all...</string>
   </property>
  </action>
  <action name="actionExport_selected">
   <property name="text">
    <string>Export selected...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
